data/tlg0006/tlg006/tlg0006.tlg006.perseus-grc2.xml

```diff
@@ -0,0 +1,2532 @@
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"  encoding='UTF-8'?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader xml:lang="eng">
+      <fileDesc>
+         <titleStmt>
+            <title type="work" n="Andr.">Andromache</title>
+            <title type="sub">Machine readable text</title>
+            <author n="Eur.">Euripides</author>
+            <editor role="editor" n="Murray">Gilbert Murray</editor>
+            <sponsor>Perseus Project, Tufts University</sponsor>
+            <principal>Gregory Crane</principal>
+            <respStmt>
+               <resp>Prepared under the supervision of</resp>
+               <name>Gregory Crane</name>
+               <name>Alison Babeu</name>
+               <name>Lisa Cerrato</name>
+            </respStmt>
+         </titleStmt>
+         <extent>About 100Kb</extent>
+         <publicationStmt>
+            <publisher>Trustees of Tufts University</publisher>
+            <pubPlace>Medford, MA</pubPlace>
+         </publicationStmt>
+         <sourceDesc default="false">
+            <bibl default="false">
+               <author>Euripides</author>
+               <title>Euripidis Fabulae, Vol 1</title>
+               <editor role="editor">Gilbert Murray</editor>
+               <pubPlace>Oxford</pubPlace>
```

Comment:
`<imprint>` is missing as is `<biblScope>` and `<ref target>` so this whole thing should look like


`<sourceDesc>`
        `<biblStruct>`
            `<monogr>`
                `<author>Euripides</author>`
                `<title xml:lang="lat">Euripidis Fabulae</title>`
                `<editor>Gilbert Murray</editor>`
                `<imprint>`
                    `<pubPlace>Oxford</pubPlace>`
                    `<publisher>Clarendon Press</publisher>`
                    `<date>1902</date>`
                `</imprint>`
               ` <biblScope unit="volume">1</biblScope>`
            `</monogr>`
           ` <ref target="https://archive.org/details/euripidisfabulae01euriuoft/page/n269/mode/2up">The Internet Archive</ref>`
       ` </biblStruct>`
    `</sourceDesc>`